cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -606,6 +614,8 @@ pub mod bridging {
 			(StartsWith<WestendEcosystem>, StartsWith<EthereumEcosystem>),
 			AssetHubWestend,
 		>;
+		pub type NonSystemParachain =
+			matching::RemoteAssetFromLocation<StartsWith<WestendEcosystem>, RandomParaLocation>;
```

Comment:
@EleisonC please, fix-and-remove those `NonSystemParachain` stuff as I mentioned here: https://github.com/paritytech/polkadot-sdk/pull/7321/files#r2163183216
```suggestion
```